infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -94,10 +96,10 @@
     private Future failureDetectorFuture = null;
     private AtomicBoolean recovered = new AtomicBoolean(false);
 
-    @Override
-    public boolean isServerReady() {
-        return true;
-    }
+    @Getter
+    private volatile boolean sequencerBootstrapped = false;
+    private final int sequencerBootstrapRetries = 3;
+    private final long sequencerBootstrapRetryTimeout = 1000;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)